jobs/jobs_utils.py

```diff
@@ -46,3 +52,27 @@ def clone_model(model, **new_values):
     props = {k: v.__get__(model, cls) for k, v in cls._properties.items()} # pylint: disable=protected-access
     props.update(new_values)
     return cls(id=model_id, **props)
+
+
+def get_model_kind(item):
+    """Returns the "kind", a globally unique identifier, of the given item.
+
+    Args:
+        item: base_models.Model|cloud_datastore.Entity|beam_datastore.Entity.
```

Comment:
What is the difference between cloud_datastore and beam_datastore (what is the "beam datastore")? (I'm wondering a bit whether the to_client_entity() call should be done here or by the caller.)